WPMedia/feed-components
# Themes 926

## Description

This PR adds the text-output-block to the feed-components repo

## Jira Ticket

- [THEMES-926](https://arcpublishing.atlassian.net/browse/THEMES-926)

## Acceptance Criteria

- text output is available in feed-components repo

## Test Steps

1. Checkout this branch `git checkout THEMES-926`
2. In the demo-outboundfeeds repo, run `npx fusion start -f -l @wpmedia/textfile-block,@wpmedia/text-output-block`
3. Verify that the text output block works correctly

[THEMES-926]: https://arcpublishing.atlassian.net/browse/THEMES-926?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
